### PR TITLE
`menu`: remove is from props

### LINKED ISSUE #3594: Rename `props`: `is*` eliminates all `is` prefix in the `menu` package
### Description

Change and unify the names of props.
Removing all is prefix from is*

### Problem Statement/Justification

For example, `isOpen` becomes `open` and `isDisabled` becomes `disabled`.

### Proposed Solution or API

props to be changed are marked `@deprecated` in v1

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by nathanlao**
Closes #3594 

## Description

Remove is prefix

## Current behavior (updates)

Before: isDisabled, isFocusable, isChecked

## New behavior

After: disabled, focusable, checked

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by Itsuki54 — DISMISSED** [review]
LGTM

File: packages/components/menu/src/menu-button.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const MenuButton = forwardRef<MenuButtonProps, "button">(
     const { onKeyDownRef, onUpstreamRestoreFocus } = useUpstreamMenuItem() ?? {}
     const {
       buttonRef,
-      isOpen,
+      isOpen: open,
```
Comment (hirotomoyamada) [inline packages/components/menu/src/menu-button.tsx:26]: The `useMenu` is a hook used internally. So, `isOpen: open` is unnecessary. Please use `open`.

File: stories/components/overlay/menu.stories.tsx
Comment (hirotomoyamada) [inline stories/components/overlay/menu.stories.tsx:1]: Don't change.

File: packages/components/menu/tests/menu.test.tsx
Comment (hirotomoyamada) [inline packages/components/menu/tests/menu.test.tsx:1]: Don't change.

File: packages/components/menu/src/menu-item.tsx [hunk 2]
```diff
@@ -71,18 +87,23 @@ export const MenuItem = forwardRef<MenuItemProps, "div">(
...
   ) => {
+    disabled ??= isDisabled
+    focusable ??= isFocusable
+
     const {
       closeOnSelect,
       focusedIndex,
-      isNested,
-      isOpen,
+      isNested: nested,
```
Comment (hirotomoyamada) [inline packages/components/menu/src/menu-item.tsx:105]: https://github.com/yamada-ui/yamada-ui/pull/3693/files#r1867394638

Comment (nathanlao) [reply packages/components/menu/src/menu-item.tsx]: Fixed in the commit [`06e6781` (#3693)](https://github.com/yamada-ui/yamada-ui/pull/3693/commits/06e6781b87f82567d837cec6ee04ab4a8714a944) !

File: packages/components/menu/src/menu-button.tsx (hunk 1 above)
Comment (nathanlao) [reply packages/components/menu/src/menu-button.tsx]: Fixed in the commit [`06e6781` (#3693)](https://github.com/yamada-ui/yamada-ui/pull/3693/commits/06e6781b87f82567d837cec6ee04ab4a8714a944) !

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**